app/Http/Controllers/SearchController.php

```diff
@@ -47,6 +48,8 @@ public function search(Request $request, $search_type = false)
             $results = $tabs = null;
         }
 
+        Log::info('Search request has been done', ['search' => $q]);
```

Comment:
Try to pass the information whenever the logged or non-logged user searched, use ```auth()->check()```.